README.md

```diff
@@ -133,6 +133,7 @@ Usage: mix coveralls <Options>
     -o (--output-dir)   Write coverage information to output dir.
     -u (--umbrella)     Show overall coverage for umbrella project.
     -v (--verbose)      Show json string for posting.
+    --flagname          coveralls.io 'flag_name' option (See coveralls.io API Reference)
```

Comment:
I assume the `coveralls.io API Reference` is about the following, but I am wondering that the meaning of the option is not clear enough.
- https://docs.coveralls.io/api-reference

How do you think about adding description like `Job flag name which will be shown in the Coveralls UI`? (if it makes sense)

- https://github.com/marketplace/actions/coveralls-github-action

Reply:
Makes sense :) Will be sorting this out today.